RELEASE.md

```diff
@@ -111,9 +111,9 @@ https://github.com/cilium/hubble/releases/new
 
 ## Finally, upload release tarballs to the GitHub release
 
-Generate the release tarballs using the `release` make target:
+Generate the release tarballs using `contrib/scripts/release.sh` script:
 
-    make release
+    ./contrib/scripts/release.sh
```

Comment:
imo `make release` should run the docker build. current release can become some other target

i find it easier to run `make release` than to remember what script and where should be run (especially if we refactor it again)

Reply:
fair point, will address in a follow up pr.

Reply:
https://github.com/cilium/hubble/pull/297